docs/migration.md

```diff
@@ -779,29 +779,30 @@ For `ConstrainedStr` you can use [`StringConstraints`][pydantic.types.StringCons
 
 ## Deprecated and moved in Pydantic V2
 
-| Pydantic V1 | Pydantic V2 |
-| --- | --- |
-| `pydantic.tools.schema_of` | `pydantic.deprecated.tools.schema_of` |
-| `pydantic.tools.parse_obj_as` | `pydantic.deprecated.tools.parse_obj_as` |
-| `pydantic.tools.schema_json_of` | `pydantic.deprecated.tools.schema_json_of` |
-| `pydantic.json.pydantic_encoder` | `pydantic.deprecated.json.pydantic_encoder` |
-| `pydantic.validate_arguments` | `pydantic.deprecated.decorator.validate_arguments` |
-| `pydantic.json.custom_pydantic_encoder` | `pydantic.deprecated.json.custom_pydantic_encoder` |
-| `pydantic.json.timedelta_isoformat` | `pydantic.deprecated.json.timedelta_isoformat` |
-| `pydantic.decorator.validate_arguments` | `pydantic.deprecated.decorator.validate_arguments` |
-| `pydantic.class_validators.validator` | `pydantic.deprecated.class_validators.validator` |
+| Pydantic V1                                | Pydantic V2                                           |
+|--------------------------------------------|-------------------------------------------------------|
+| `pydantic.tools.schema_of`                 | `pydantic.deprecated.tools.schema_of`                 |
+| `pydantic.tools.parse_obj_as`              | `pydantic.deprecated.tools.parse_obj_as`              |
+| `pydantic.tools.schema_json_of`            | `pydantic.deprecated.tools.schema_json_of`            |
+| `pydantic.json.pydantic_encoder`           | `pydantic.deprecated.json.pydantic_encoder`           |
+| `pydantic.validate_arguments`              | `pydantic.deprecated.decorator.validate_arguments`    |
+| `pydantic.json.custom_pydantic_encoder`    | `pydantic.deprecated.json.custom_pydantic_encoder`    |
+| `pydantic.json.timedelta_isoformat`        | `pydantic.deprecated.json.timedelta_isoformat`        |
+| `pydantic.json.ENCODERS_BY_TYPE`           | `pydantic.deprecated.json.ENCODERS_BY_TYPE`           |
```

Comment:
Looks good overall, but no need to change the formatting. Should be a 1 line diff. Happy to accept once you change things back 👍

Reply:
Oh my bad, it was inadvertent. Will correct it now.